packages/vue-dot/src/elements/PageContainer/PageContainer.vue

```diff
@@ -0,0 +1,68 @@
+<template>
+	<VSheet
+		class="vd-page-container"
+		:class="paddingClass"
+		:width="containerSize"
+	>
+		<slot />
+	</VSheet>
+</template>
+
+<script lang="ts">
+	import Vue from 'vue';
+	import Component, { mixins } from 'vue-class-component';
+
+	import { IndexedObject } from '../../types';
+
+	const Props = Vue.extend({
+		props: {
+			/**
+			 * Define the size of the container
+			 *
+			 * Default value: "xl"
+			 */
+			size: {
+				type: String,
+				validator: value => [
+					'xl',
+					'l',
+					'm',
+					'xs'
+				].includes(value.toLowerCase()),
+				default: 'xl'
+			}
+		}
+	});
+
+	const MixinsDeclaration = mixins(Props);
+
+	@Component
+	export default class PageContainer extends MixinsDeclaration {
+		/**
+		 * Use vutify breackpoint to return responsive margin for the container
+		 */
+		get paddingClass(): string {
+			const breakpoints : IndexedObject<string> = {
+				'xs': 'mx-0',
+				'sm': 'mx-4',
+				'md': 'mx-8',
+				'lg': 'mx-8',
+				'xl': 'mx-8'
+			};
+			return breakpoints[this.$vuetify.breakpoint.name];
```

Comment:
```suggestion
			const containerMargin: IndexedObject<string> = {
				xs: 'mx-0',
				sm: 'mx-4',
				md: 'mx-8',
				lg: 'mx-8',
				x': 'mx-8'
			};

			const margin = containerMargin[this.$vuetify.breakpoint.name];

			return `my-10 mx-${margin}`;
```
J'ai renommé `breakpoints` en `containerMargin` pour être plus cohérent avec le getter `containerSize` et surtout j'ai ajouté `my-10` pour l'espacement vertical !